src/index.js

```diff
@@ -43,21 +44,15 @@ class IpfsRepo {
   constructor (repoPath, options) {
     assert.equal(typeof repoPath, 'string', 'missing repoPath')
 
-    if (options == null) {
-      options = require('./default-options')
```

Comment:
Since webpack and browserify extract dependencies using static analysis, this is always going to get included in the bundle, with or without these lines.